### ISSUE TITLE
[BUG] "from __future__ imports must occur at the beginning of the file" when using multiline comment above

### ISSUE THREAD

**Issue by EmberLightVFX** [dashingsoft/pyarmor#1587]
If I have a multiline-comment in the top of my code where I also use `from __future__ import`, mix-str causes an error:
`from __future__ imports must occur at the beginning of the file`

Here's an example code:
```
"""
Some comment
that uses multiple lines
"""
from __future__ import annotations

from datetime import datetime
from typing import TYPE_CHECKING

from pydantic import BaseModel
if TYPE_CHECKING:
    from typing import Dict, List


class SomeModel(BaseModel):
    some_str: str
    created_at: datetime
    mapping: Dict[str, List[int]]
```

If I change the code to use normal commas instead it works for me:
```
# --------------------
# Some comment
# that uses multiple lines
# --------------------
from __future__ import annotations

from datetime import datetime
from typing import TYPE_CHECKING

from pydantic import BaseModel
if TYPE_CHECKING:
    from typing import Dict, List


class SomeModel(BaseModel):
    some_str: str
    created_at: datetime
    mapping: Dict[str, List[int]]
```

```
INFO     Python 3.10.11
INFO     Pyarmor 8.4.3 (group), 006141, aesl
INFO     Platform windows.x86_64
```

**Comment by jondy** [comment]
Reproduced, it will be fixed in next release.

**Comment by jondy** [comment]
Fixed in v8.4.4
